### DISCUSSION TITLE
Design token file availability

### DISCUSSION THREAD

**Discussion by nyan-matt** [primefaces/community#3251] [Figma UI Kit] [answered]
I'm helping evaluate ways to integrate an existing set of design tokens with the new v18 theming changes. My existing token set mostly consists of primitive, semantic, and a few component level tokens. My next step was to try to inventory the existing token names and structure and map that into Prime's architecture and naming scheme. 

Is there a json file available (compatible w/ style dictionary or new w3c dtcg draft spec) that represents the token data as represented in your [UI kit Tokens page](https://www.figma.com/design/G855HjuSyK8viJr0a5ZjRG/Preview-%7C%C2%A0PrimeOne-%7C-3.0.1?node-id=3401-29521)

Thanks!

**Comment by onursenture** [comment, marked as answer]
Hi, We cannot publicly share the tokens.json file because it's part of a paid product. If you purchase PrimeOne, you can obtain it using the export feature of the Tokens Studio plugin.
